### ISSUE TITLE
What is this tool?

### ISSUE THREAD

**Issue by karliky** [guillaumeguerin/Gw2Machinima#1]
I'm not sure if you've seen my machinima videos, but I'm working in a machinima tool too

**Comment by guillaumeguerin** [comment]
Yes, I saw a beautiful machinima video recently on reddit wich gave me the idea for this project. 
Maybe it was yours :)

I have very little knowledge of Reverse Engeneering though, but I already made Zoom hacks for Guild Wars 1 & 2. I need to improve them also as you can see in that repo : https://github.com/guillaumeguerin/GwZoomHack

**Comment by karliky** [comment]
Yes! That was my video :) you can find my machinima tool here https://github.com/karliky/machinima-studio

would be great if you want to join my project, I have a lot of things to do!

**Comment by guillaumeguerin** [comment]
Thank you for sharing this with me :)
I'm going to fork this project and play with it !
